src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Microsoft.Azure.Commands.ApiManagement.ServiceManagement.dll-help.Help.pshproj

```diff
@@ -5894,7 +5894,7 @@ New-AzureRmApiManagementOperation –Context $apimContext –ApiId $apiId –Ope
       </ParamSets>
       <Syntax>
```

Comment:
This is no longe rused for help, please remove.  Instead, follow the instructions here:  https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md for creating and using the platyPS tool for markdown files.